nativelib/src/main/scala/scala/scalanative/runtime/DeleteOnExit.scala

```diff
@@ -0,0 +1,22 @@
+package scala.scalanative.runtime
+
+import scala.collection.mutable
+import scala.scalanative.native.{Zone, stdio, toCString}
+
+object DeleteOnExit {
+  private[this] val toDeleteSet: mutable.Set[String] = mutable.Set.empty
+  private[this] val toDelete: mutable.ArrayBuffer[String] =
```

Comment:
Just `private` should be sufficient.

Reply:
Yeah, I guess in this particular case there is no reason for private[this]. It's just a habit of mine.